mutis/lib/signal.py

```diff
@@ -83,9 +83,9 @@ def lc_gen_psd_lombscargle(times, values):
         sigp = values
         tp = times
 
-    N = values.size
-    # k = np.arange(-N/2, N/2) no bc scipy_signal.lombscargle does not support freq zero
-    k = np.linspace(-N / 2, N / 2 - 1 + 1e-6, N)
+    n = values.size
+    # k = np.arange(-n/2, n/2) no bc scipy_signal.lombscargle does not support freq zero
+    k = np.linspace(-n / 2, n / 2 - 1 + 1e-6, n)
```

Comment:
is the `1e-6` related to time/frequency? Then I'd suggest to define that as a variable like

Reply:
done

Reply:
it is not a 'scale', it is just an offset so there is no zero frequency and the function gives an error... maybe call it offset instead of scale, if you want it to have a name (I would not)?

Reply:
I have renamed the var to `offset`